src/classes/OrgConfig.cls

```diff
@@ -63,6 +76,10 @@ public with sharing class OrgSettings {
         private set;
     }
 
+    public Boolean isStateCountryPicklistsEnabled() {
```

Comment:
Why still static?

Reply:
Should be fixed now!  About to push up one more round of changes then will stop for final review!